provisioning/utils.go

```diff
@@ -107,6 +107,62 @@ func GetIronicIP(client kubernetes.Interface, targetNamespace string, config *me
 	return
 }
 
+func GetPodIP(client kubernetes.Interface, targetNamespace string, networkType NetworkStackType) (string, error) {
```

Comment:
This is essentially a copy of getPodHostIP, could these be refactored into one call?